Cython/TestUtils.py

```diff
@@ -253,4 +275,4 @@ def write_newer_file(file_path, newer_than, content, dedent=False):
         other_time = None
 
     while other_time is None or other_time >= os.path.getmtime(file_path):
-        write_file(file_path, content, dedent=dedent)
+        os.utime(file_path, None)
```

Comment:
I had to look up the second argument in the documentation to understand what this does. Let's use at least a keyword argument (as should always be done for parameters like `None`, `False`, and the like).
```suggestion
        os.utime(file_path, times=None)
```
Alternatively, a helper function `touch_file(file_path)` would probably be a helpful addition.

Reply:
> helper function touch_file(file_path) would probably be a helpful addition

Good idea, I'll add it.

Reply:
`TypeError: utime() takes no keyword arguments` in python 2.7, ironically

Reply:
[resolved](https://github.com/cython/cython/pull/4241/commits/8e65bcfeb364b332b8fb7aa5153026cac43205ff)